src/SFA.DAS.EmployerFinance.Web/Startup/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using SFA.DAS.EmployerFinance.Configuration;
+using SFA.DAS.EmployerFinance.Configuration.AzureTableStorage;
+using NLog.Web;
+using StructureMap.AspNetCore;
+
+namespace SFA.DAS.EmployerFinance.Web.Startup
```

Comment:
I usually like to see an entry point into an application in the root, what do you think about moving `Program.cs` back to the root but keep the `Startup.cs` files here?

Reply:
I've moved the `NServiceBusStartup` classes into `Startup` folders/namespaces.

I've gotten used to finding the startup classes in one folder, including the `Program` class - it's nice for them to be clustered together when working on the web's startup. Also the root of the web project is currently somewhat cluttered with miscellaneous files (mostly json config files).

Saying that, if you really want `Program.cs` in the root, I'll put it back there.